### PR TITLE
Implement oms3_setReal/oms3_getReal

### PR DESCRIPTION
**Pull request by lochel**
### Purpose

Implement oms3_setReal/oms3_getReal

### Type of Change

- Code refactoring (non-breaking change which improves maintainability)
- New feature (non-breaking change which adds functionality)

### Checklist

- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by lochel** [comment]
This works now for connectors and CS FMUs. However, the variable lookup need to be improved and the SSP import/export is missing.

File: src/OMSimulatorLib/System.h [hunk 1]
```diff
@@ -108,6 +108,12 @@ namespace oms3
     virtual oms_status_enu_t terminate() = 0;
     virtual oms_status_enu_t stepUntil(double stopTime) = 0;
 
+    oms_status_enu_t getReal(const ComRef& cref, double& value) const;
+    oms_status_enu_t setReal(const ComRef& cref, double value);
+    oms_status_enu_t getReals(const std::vector<ComRef> &crefs, std::vector<double> &values) const;
+    oms_status_enu_t setReals(const std::vector<ComRef> &crefs, std::vector<double> values);
+    oms_status_enu_t setRealInputDerivatives(const ComRef &cref, int order, double value);
```
Comment (lochel) [inline src/OMSimulatorLib/System.h]: @robbr48 I moved `getReal`/`setReal` to `class System` to implement it only once for all system kinds. I moved your methods as well, but I am not certain about `setRealInputDerivatives`. Is it for system inputs or component (FMU?) inputs? Or for both?

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being [tested](https://test.openmodelica.org/hudson/job/OMSimulator-PR/754//console). It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#445
  - OpenModelica/OMSimulator-testsuite#105

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
All tests were successful. :confetti_ball:

File: src/OMSimulatorLib/System.h (hunk 1 above)
Comment (robbr48) [reply src/OMSimulatorLib/System.h]: I guess it needs to be propagated through the connections in the same way as setReal.

**Pull request closed without merging**